A VecCheckNan issue (#491)
hill-a/stable-baselines: docs/misc/changelog.rst

  `Issue #430 <https://github.com/hill-a/stable-baselines/issues/430>`_.
- Fix a bug when calling `logger.configure()` with MPI enabled (@keshaviyengar)
- set `allow_pickle=True` for numpy>=1.17.0 when loading expert dataset

> - Fix a bug when use VecCheckNan to check env and use numpy ndarray as state.  `Issue #489 <https://github.com/hill-a/stable-baselines/issues/489>`_.

Reviewer: Please at your username + at the bottom too ;)

Author: Thank you very much. It's my first to create a PR and I'm a little excited to do something for such a big project. :)

> - Fix a bug when use VecCheckNan to check env and use numpy ndarray as state.  `Issue #489 <https://github.com/hill-a/stable-baselines/issues/489>`_.

Reviewer: ```suggestion
- Fix a bug when using VecCheckNan with numpy ndarray as state.  `Issue #489 <https://github.com/hill-a/stable-baselines/issues/489>`_.
```
